org.eclipse.jdt.core/dom/org/eclipse/jdt/core/dom/GuardedPattern.java

```diff
@@ -193,9 +193,9 @@ public static List propertyDescriptors(int apiLevel, boolean previewEnabled) {
 
 	/**
 	 * Returns the conditional expression of this pattern, or
-	 * <code>null</code> if there is none (the "default:" case).
+	 * <code>this.ast.newNullLiteral()</code> if there is none (the "default:" case).
```

Comment:
Please put a link to NullLiteral  {@link NullLiteral} instead of this.ast.newNullLiteral() - modify in other placess also

Reply:
Done